strings/camel_case_to_snake_case.py

```diff
@@ -0,0 +1,83 @@
+"""
+this function focuses on converting camel case (eg: camelCase)
+to snake case (snake_case)
+"""
+
+
+def to_snake_case(word: str) -> str:
+    """
+    This to_snake_case function convert camel case words (eg: camelCase)
+    to snake case (eg: snake_case)
+    >>> to_snake_case('helloWorld')
+    'hello_world'
+    >>> to_snake_case('camelCase')
+    'camel_case'
+    >>> to_snake_case('hacktoberFest')
+    'hacktober_fest'
+    >>> to_snake_case('')
+    Traceback (most recent call last):
+    Exception: Provide camel case eg: helloWorld
+    >>> to_snake_case('HelloWorld')
+    Traceback (most recent call last):
+    Exception: Provide camel case eg: helloWorld
+    >>> to_snake_case(' ')
+    Traceback (most recent call last):
+    Exception: Provide camel case eg: helloWorld
+    """
+
+    """
+    Raising the error if inputted data is not in the required format
+    """
+
+    # checks if the word has capital character in it,
+    # and first character is not the capital.
+    if word != word.lower() and word[0] == word[0].lower():
+        snake_case = ""
+        index_list = []
+        # check number of Capital letter and append their indexes in list
+        for character in word:
+            if character != character.lower():
+                index_list.append(word.index(character))
+
+        # append one more element
```

Comment:
Why?

Reply:
I am appending the indexes of the uppercase characters. I will need to add an underscore ( _ ) before it.